### PR TITLE
Re-styled the online manual. Pretty basic stuff, but looks much nicer now.

### PR DESCRIPTION
**Pull request by Void-Frost**
https://i.imgur.com/V29dGoO.png

### CODE REVIEW

**Review by NightYoshi370 — APPROVED** [review]
While I do think this looks basic and could be looking a bit more like the rest of the manual pages, this looks much nicer than the one we have right now. I might make my own take, but not anytime soon so I approve this for now.

**Comment by Void-Frost** [comment]
> While I do think this looks basic and could be looking a bit more like the rest of the manual pages, this looks much nicer than the one we have right now. I might make my own take, but not anytime soon so I approve this for now.

Do you have suggestions on how to make it look more like the rest of the manual pages? I'm not really sure how else, I copied the color scheme.

Any suggestions are totally appriciated.

**Comment by Epicpkmn11** [comment]
スクリーンショット 2021-01-05 18 58 03

Maybe something like this could work if you want it to look like the pages themselves. That's just a quick edit of one of the actual pages with the sections like this:
```html
<div class="section-title">Dansk</div>
<div class="section-body">
	<ul>
		<li><a href="/pages/danish/_howToUse.html">How to use</a></li>
		<li><a href="/pages/danish/chat.html">Chat</a></li>
		<li><a href="/pages/danish/gbarunner2.html">GBARunner2</a></li>
		<li><a href="/pages/danish/nds-bootstrap.html">nds-bootstrap</a></li>
		<li><a href="/pages/danish/settings.html">Settings</a></li>
		<li><a href="/pages/danish/theme1_dsi.html">DSi theme</a></li>
		<li><a href="/pages/danish/theme2_3ds.html">3DS theme</a></li>
		<li><a href="/pages/danish/theme3_r4.html">Original R4 theme</a></li>
		<li><a href="/pages/danish/theme4_acekard.html">Wood UI theme</a></li>
		<li><a href="/pages/danish/theme5_saturn.html">SEGA Saturn theme</a></li>
		<li><a href="/pages/danish/theme6_hbl.html">Homebrew Launcher theme</a></li>
	</ul>
</div>
```

File: index.html [hunk 1]
```diff
@@ -1,278 +1,279 @@
-<!DOCTYPE html>
+﻿<!DOCTYPE html>
 <html lang="en">
 <head>
+	<link rel="stylesheet" href="/style.css">
 	<meta charset="utf-8">
```
Comment (Epicpkmn11) [inline index.html:5]: Also, minor thing but this should always be the first thing in the head ;P

**Comment by NightYoshi370** [comment]
Do you think you could implements Evie's suggestion? That's what I wanted to do, but since there's already a PR, might as well :P

**Comment by Void-Frost** [comment]
Ok, so what I'm going to do, is close this PR, work more, then create a new one.

**Pull request closed without merging**